_docs/concepts/configuration/model.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Data model
+overview: Istio configuration data model.
+
+order: 10
+
+layout: docs
+type: markdown
+---
+
+Istio config is a conglomerate of config aspects. Each config aspects is
+corresponding to an kubernetes resource. The organization of resources is "flat"
+on files representation. The Kubernetes API server stores the config data
+according to name and namespace in the hierarchy.
+
+Istio config is completely built on Kubernetes resources. Besides a small set of
```

Comment:
Saying "Kubernetes resource" here could confuse with k8s core resources. Let's change to k8s CRD?